WalletWasabi/Blockchain/Keys/HdPubKey.cs

```diff
@@ -23,7 +23,8 @@ public HdPubKey(PubKey pubKey, KeyPath fullKeyPath, SmartLabel label, KeyState k
 		PubKey = Guard.NotNull(nameof(pubKey), pubKey);
 		FullKeyPath = Guard.NotNull(nameof(fullKeyPath), fullKeyPath);
 		_cluster = new Cluster(this);
-		SetLabel(label, null);
+		Label = label ?? SmartLabel.Empty;
```

Comment:
It cannot be null as label is not nullable. Check if we assign label to null somewhere in that case you might need `SmartLabel?`